lib/src/__tests__/DateRangePicker.test.tsx

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
-import { DesktopDateRangePicker } from '../';
+import { isWeekend } from 'date-fns';
 import { screen, waitFor } from '@testing-library/react';
 import { utilsToUse, getAllByMuiTest } from './test-utils';
 import { TextField, TextFieldProps } from '@material-ui/core';
+import { DesktopDateRangePicker, StaticDateRangePicker } from '../';
```

Comment:
```suggestion
import { DesktopDateRangePicker, StaticDateRangePicker } from '@material-ui/picker';
```

Reply:
No it doesn’t work locally for some reason. Need investigation